app/controllers/cases/offender_sar_controller.rb

```diff
@@ -212,15 +212,38 @@ def confirm_sent_to_sscl
       redirect_to case_path(@case) and return
     end
 
-    def confirm_information_received
-      information_received = params[:offender_sar].try(:[], :information_received)
-      return if information_received.present?
+    def accepted_date_received
+      authorize @case, :can_edit_case?
+      authorize @case, :can_validate_rejected_case?
```

Comment:
Rather than calling making 2 authorisation checks, could `can_validate_rejected_case?` also check the user can edit the case?